### PR TITLE
Release notes and installation updates for v0.1.16

### PR DESCRIPTION
**Pull request by CharlieSYH**
## Info

- **Preview**: 
[ Paste the preview link to the edited page here. ]

Please update the release date before merging.

## Before merging
  - [ ] (For version-specific PR) I have selected the corresponding software version in **Milestone** and linked the related doc issue to this PR in **Development**.
  - [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`bernscode`, `CharlieSYH`, `emile-00`, & `hengm3467`). 
  - [ ] I have checked the doc site preview, and the updated parts look good. <details><summary>How?</summary>Scroll down and open this link: image
</details>

### CODE REVIEW

**Review by huangjw806 — APPROVED** [review]
LGTM!

File: docs/release-notes.md [hunk 1]
```diff
@@ -7,6 +7,46 @@ slug: /release-notes
...
+- Breaking change: Deprecates `CREATE MATERIALIZED SOURCE` . To create a materialized source, create a table and include the newly added connector settings. [#7110](https://github.com/risingwavelabs/risingwave/pull/7281https://github.com/risingwavelabs/risingwave/pull/7110)
+- Adds support for the `c` and `i` flags in `regex_match()` and `regex_matches()` functions. [#7135](https://github.com/risingwavelabs/risingwave/pull/7135)
+- Adds support for `SHOW CREATE TABLE` . You can use this statement to show the definition of a table. [#7152](https://github.com/risingwavelabs/risingwave/pull/7152)
+- Adds support for the `pg_stat_activity` system catalog and several system functions. [#7274](https://github.com/risingwavelabs/risingwave/pull/7274)
+- Adds the `_rw_kafka_timestamp` parameter to show the timestamps of Kafka messages. Users can now specify the scope of Kafka messages by timestamps. [#7275](https://github.com/risingwavelabs/risingwave/pull/7275), [#7150](https://github.com/risingwavelabs/risingwave/pull/7150)
+- Adds support for display PostgreSQL and RisingWave version in `version()`. [#7314](https://github.com/risingwavelabs/risingwave/pull/7314)
```
Comment (hengm3467) [inline docs/release-notes.md]: display... version -> displaying ... versions @CharlieSYH

File: docs/release-notes.md [hunk 2]
```diff
@@ -7,6 +7,46 @@ slug: /release-notes
...
+
+#### Administration
+
+- Adds support for aborting a query in local mode with `Ctrl + C`. [#7444](https://github.com/risingwavelabs/risingwave/pull/7444)
+
+#### SQL features
+
+- Adds support for the `to_timestamp` function. [#7060](https://github.com/risingwavelabs/risingwave/pull/7060)
+- Adds support for the `RETURNING` clause in DML statements. [#7094](https://github.com/risingwavelabs/risingwave/pull/7094)
+- Breaking change: Deprecates `CREATE MATERIALIZED SOURCE` . To create a materialized source, create a table and include the newly added connector settings. [#7110](https://github.com/risingwavelabs/risingwave/pull/7281https://github.com/risingwavelabs/risingwave/pull/7110)
```
Comment (hengm3467) [inline docs/release-notes.md]: There are two related PRs in this row, but only 7110 is shown as a link.

**Review by hengm3467 — CHANGES_REQUESTED** [review]
Please implement the two minor issues before merging the changes. Thanks. @CharlieSYH

**Review by hengm3467 — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**